<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDlg</class>
 <widget class="QDialog" name="PreferencesDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>420</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QListWidget" name="categoriesListBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>511</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="generalPage">
    <widget class="QGroupBox" name="colorGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Color</string>
     </property>
     <widget class="QRadioButton" name="blackRadio">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Black</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="redRadio">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Red</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="greyRadio">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Grey</string>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="searchCheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>261</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>If checked, typing &quot;ame&quot; in library search would find designs with the keyword &quot;america&quot;, but not &quot;cameo&quot;. If not checked, both would be found.</string>
     </property>
     <property name="text">
      <string>Library search matches only the start of words.</string>
     </property>
    </widget>
    <widget class="QLabel" name="searchExplanationLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>If checked, typing &quot;ami&quot; in library search would find files with the keyword &quot;animal&quot;, but not &quot;panic&quot;. If not checked, both would be found.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="displayPage">
    <widget class="QGroupBox" name="antialiasingGroup">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>201</width>
       <height>271</height>
      </rect>
     </property>
     <property name="title">
      <string>Antialiasing</string>
     </property>
     <widget class="QRadioButton" name="aaOffRadio">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Off</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="aaBasicRadio">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Basic</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="aa2xRadio">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>2x Samples</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="aa4xRadio">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>4x Samples</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="aa6xRadio">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>6x Samples</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="aa8xRadio">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>8x Samples</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="editingPage"/>
   <widget class="QWidget" name="languagePage"/>
   <widget class="QWidget" name="advancedPage"/>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
